### PR TITLE
Remove historical roots

### PR DESCRIPTION
**Pull request by gazzua**
- Remove historical Roots
- Fix tests
- Fix broken Partial Withdrawl Test

### CODE REVIEW

File: config/params/loader_test.go [hunk 1]
```diff
@@ -114,7 +114,7 @@ func assertEqualConfigs(t *testing.T, name string, fields []string, expected, ac
 	assert.Equal(t, expected.SlotsPerHistoricalRoot, actual.SlotsPerHistoricalRoot, "%s: SlotsPerHistoricalRoot", name)
 	assert.Equal(t, expected.MinValidatorWithdrawabilityDelay, actual.MinValidatorWithdrawabilityDelay, "%s: MinValidatorWithdrawabilityDelay", name)
 	assert.Equal(t, expected.ShardCommitteePeriod, actual.ShardCommitteePeriod, "%s: ShardCommitteePeriod", name)
-	assert.Equal(t, expected.MinEpochsToInactivityPenalty, actual.MinEpochsToInactivityPenalty, "%s: MinEpochsToInactivityPenalty", name)
+	assert.Equal(t, expected.MinEpochsToInactivityPenalty, actual.MinEpochsToInactivityPenalty, "%s: MinEpxochsToInactivityPenalty", name)
```
Comment (syjn99) [inline config/params/loader_test.go]: Gotcha

File: beacon-chain/state/state-native/state_trie.go [hunk 2]
```diff
@@ -111,12 +108,12 @@ var electraFields = append(
...
-	bellatrixSharedFieldRefCount                  = 11
-	capellaSharedFieldRefCount                    = 12
-	denebSharedFieldRefCount                      = 12
-	electraSharedFieldRefCount                    = 14
+	phase0SharedFieldRefCount                     = 8
+	altairSharedFieldRefCount                     = 9
+	bellatrixSharedFieldRefCount                  = 10
+	capellaSharedFieldRefCount                    = 11
+	denebSharedFieldRefCount                      = 11
+	electraSharedFieldRefCount                    = 13
```
Comment (syjn99) [inline beacon-chain/state/state-native/state_trie.go:114]: Seems like `experimental*` constants should be decremented also

File: config/params/loader_test.go (hunk 1 above)
Comment (gazzua) [reply config/params/loader_test.go]: 🥲

**Review by schyun9212 — APPROVED** [review]
LGTM

**Pull request merged**